packages/share/ios/Classes/SharePlugin.m

```diff
@@ -15,8 +15,30 @@ + (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar> *)registrar {
 
   [shareChannel setMethodCallHandler:^(FlutterMethodCall *call, FlutterResult result) {
     if ([@"share" isEqualToString:call.method]) {
+      NSDictionary *arguments = [call arguments];
+
+      if ([arguments[@"text"] length] == 0) {
+        result([FlutterError errorWithCode:@"ARGUMENT_ERROR"
```

Comment:
You may want to use `@"error"` for consistency with how the Java side works with uncaught RuntimeExceptions.

Reply:
I'm explicitly calling `result.error("ARGUMENT_ERROR", ...` on the Java side. Did you mean runtime exceptions end up equating @"error"?

Reply:
Yes. IIRC

Reply:
https://github.com/flutter/engine/blob/master/shell/platform/android/io/flutter/plugin/common/MethodChannel.java#L209

Reply:
Done